libsignal-protocol/Cargo.toml

```diff
@@ -13,3 +13,6 @@ failure_derive = "0.1.5"
 rand = "0.6.5"
 parking_lot = "0.8.0"
 lock_api = "0.2.0"
+
+[target.'cfg(target_os = "linux")'.dependencies]
```

Comment:
What are your thoughts on putting the platform-specific backends behind their own feature flags? Linking to native C libraries can get pretty tricky and I'd prefer to make compiling the basic case as easy as possible...

Reply:
That would be great, having feature flags instead of platform specific implementation would work, since most people would want to use `openssl` or the rust `native` features.

I agree with that :+1: